src/quo2/core.cljs

```diff
@@ -0,0 +1,87 @@
+(ns quo2.core
+  (:require quo2.components.buttons.button
+            quo2.components.buttons.dynamic-button
+            quo2.components.markdown.text
+            quo2.components.icon
+            quo2.components.separator
+            quo2.components.header
+            quo2.components.avatars.account-avatar
+            quo2.components.avatars.channel-avatar
+            quo2.components.avatars.group-avatar
+            quo2.components.avatars.icon-avatar
+            quo2.components.avatars.user-avatar
+            quo2.components.avatars.wallet-user-avatar
+            quo2.components.community.community-card-view
+            quo2.components.community.community-list-view
+            quo2.components.community.community-view
+            quo2.components.community.discover-card
+            quo2.components.counter.counter
+            quo2.components.dividers.divider-label
+            quo2.components.dividers.new-messages
+            quo2.components.drawers.action-drawers
+            quo2.components.dropdowns.dropdown
+            quo2.components.info.info-message
+            quo2.components.info.information-box
+            quo2.components.list-items.channel
+            quo2.components.list-items.menu-item
+            quo2.components.list-items.preview-list
+            quo2.components.messages.gap
+            quo2.components.messages.system-message
+            quo2.components.reactions.reaction
+            quo2.components.notifications.activity-logs
+            quo2.components.notifications.info-count
+            quo2.components.notifications.notification-dot
+            quo2.components.tags.tags
+            quo2.components.tabs.tabs
+            quo2.components.tabs.account-selector))
+
+(def button quo2.components.buttons.button/button)
+(def dynamic-button quo2.components.buttons.dynamic-button/dynamic-button)
+(def text quo2.components.markdown.text/text)
+(def icon quo2.components.icon/icon)
+(def separator quo2.components.separator/separator)
+(def counter quo2.components.counter.counter/counter)
+(def header quo2.components.header/header)
+(def action-drawer quo2.components.drawers.action-drawers/action-drawer)
+(def dropdown quo2.components.dropdowns.dropdown/dropdown)
+(def info-message quo2.components.info.info-message/info-message)
+(def information-box quo2.components.info.information-box/information-box)
+(def gap quo2.components.messages.gap/gap)
+(def system-message quo2.components.messages.system-message/system-message)
+(def reaction quo2.components.reactions.reaction/reaction)
+(def tags quo2.components.tags.tags/tags)
+(def tabs quo2.components.tabs.tabs/tabs)
+(def scrollable-tabs quo2.components.tabs.tabs/scrollable-tabs)
+(def account-selector quo2.components.tabs.account-selector/account-selector)
+
+;; AVATAR
```

Comment:
Just a tiny change, in Lisps it's more idiomatic to use 3 or 4 semicolons for section-like comments. Some editors (e.g. Emacs with `outline-mode`) will take advantage of it in many useful ways. This is also recommended in the Clojure Style Guide.